File: 0167.two-sum-ii-input-array-is-sorted/two-sum-ii-input-array-is-sorted.java
class Solution {
    public int[] twoSum(int[] numbers, int target) {
        int[] index = {0, numbers.length - 1};
        while (numbers[index[0]] + numbers[index[1]] != target)
        {
            if (numbers[index[0]] + numbers[index[1]] < target) index[0]++;
            else index[1]--;
        }
        index[0]++; index[1]++;
        return index;
    }
}

File: 0167.two-sum-ii-input-array-is-sorted/two-sum-ii-input-array-is-sorted.cs
public class Solution {
    public int[] TwoSum(int[] numbers, int target) {
        int[] index = new int[2] {0,0};
        for (int i = 0;i < numbers.Length - 1; i++)
        {
            for (int j = i + 1; j < numbers.Length; j++)
            {
                if (numbers[i] + numbers[j] == target)
                {
                    index[0] = i + 1; index[1] = j + 1;
                    break;
                }
            }
        }
        return index;
    }
}